src/components/scene/stats.js

```diff
@@ -16,6 +17,9 @@ module.exports.Component = registerComponent('stats', {
 
   init: function () {
     var scene = this.el;
+
+    if (utils.getUrlParameter('stats') === 'false') { return; }
```

Comment:
> And there's not really a use case for needed `?stats=true`, but if we wanted to see them, we can attach stats via devtools.

Why would there not be a use case for them? I can imagine someone writing a script that fires up a handful of scenes to quickly look at the stats.

Just for the sake of expectations, if I see a URL with `?stats=false`, I'm going to assume that `?stats=true` turns stats on. Why would not enable both?

> We can't because this code is only run when you actually have stats attached.

Yeah, but wouldn't you just need to call this when `?stats=true`:

```js
this.el.setAttribute('stats', '');
```